### DISCUSSION TITLE
Please give me access to Copilot

### DISCUSSION THREAD

**Discussion by just1a-person** [community/community#4482] [Copilot Conversations] [answered]
please 😔

**Comment by EarthlyAlien** [comment]
Sign up for the waitlist here https://github.com/features/copilot/signup
You will not be able to use GitHub Copilot until you are off of the waitlist. Or else you have to wait till it gets into GA (general availability)

**Reply by just1a-person** [reply]
I am already on the waitlist. I had gotten access to Codespaces by asking a GitHub employee, so...

**Reply by azartheen** [reply]
i signup copilot but it almost more than 5 days I am still on the waitlist!... can i know the reason why i can't get access

**Reply by azartheen** [reply]
Hey folks!!! after 3 more months... got an access

**Reply by SmartGecko44** [reply]
i have been on the waitlist for over 7 months  so please do not complain about having to wait for 5 days

> i signup copilot but it almost more than 5 days I am still on the waitlist!... can i know the reason why i can't get access

**Comment by michellemerrill** [comment, marked as answer]
As GitHub Copilot requires state-of-the-art AI hardware, we are offering it to a limited number of testers for free, during the technical preview. We want to use the preview to learn how people use GitHub Copilot and what it takes to operate it at scale. If the technical preview is successful, our plan is to build a commercial version of GitHub Copilot in the future. To learn more about GitHub Copilot and our latest updates, please head over to https://copilot.github.com/.

**Reply by faisalaus** [reply]
I am waiting for access for over a week now. How long it takes to get access? Thanks

**Reply by YusufSuleman** [reply]
@faisalaus , it is probably a lie.

**Reply by Perhaps234** [reply]
I've been waiting for like a month now, please let me know when I can get it.

**Reply by YusufSuleman** [reply]
> I've been waiting for like a month now, please let me know when I can get it.

It is worth the wait. Just talk about VS Code around a big event like GitHub Universe.

**Comment by an0n-an9e1** [comment]
I don't thinks they will accept any more, just check the gpt-3 waitlist, its already closed!

**Comment by fatemagol** [comment]
Hello, good time. I am a senior student of parasitology and I was using Copilot for research work. I think my account was blocked by mistake. Please check. Thank you for your service.

**Comment by fatemagol** [comment]
How to fix this error?
This content is blocked. Contact the site owner to fix the issue.
